src/components/MultiHopTrade/components/TradeInput/components/Claim/hooks/useArbitrumClaimsByStatus.tsx

```diff
@@ -36,47 +34,40 @@ export type ClaimDetails = Omit<ClaimStatusResult, 'status'> & {
 
 type ClaimsByStatus = Record<ClaimStatus, ClaimDetails[]>
 
-export const useArbitrumClaimsByStatus = (txs: Tx[]) => {
+export const useArbitrumClaimsByStatus = (props?: { isDisabled?: boolean }) => {
```

Comment:
Nitpick: `skip` is conventional for skipping hooks logic run

Reply:
yessir 94acc62d7910a92f005332759950b378b8aa618c